cbmc/proofs/Makefile.common

```diff
@@ -0,0 +1,244 @@
+SHELL=/bin/bash
+
+default: report
+
+################################################################
+# Define locations and binaries
+
+#MQTT ?= ${HOME}/mqtt
+MQTT ?= $(abspath ../../..)
```

Comment:
I'd prefer not to have code/definitions commented out without further comments explaining why one of them may be useful.

Reply:
This is gone in the squashed commit.